index.d.ts

```diff
@@ -370,6 +376,13 @@ export interface CarouselProps {
    * @default false
    */
   withoutControls?: boolean;
+
+  /**
+   * Sets the RTL to default
```

Comment:
This description doesn't really make sense. The type doesn't set the `rtl` to default, this prop is used to set the direction of the slides. So this should read something more like that